modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java

```diff
@@ -985,6 +991,151 @@ private void _testObjectEntryMtoMObjectRelatedModelsProviderImpl(
 				reverseObjectRelationship.getObjectRelationshipId()));
 	}
 
+	private void _testOrganizationSystemObjectEntry1toMObjectRelatedModelsProviderImpl()
+		throws Exception {
+
+		Organization organization1 = OrganizationTestUtil.addOrganization(
+			OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID,
+			"Organization 1", false);
+
+		Organization organization2 = OrganizationTestUtil.addOrganization(
+			OrganizationConstants.DEFAULT_PARENT_ORGANIZATION_ID,
+			"Organization 2", false);
+
+		ObjectDefinition systemObjectDefinition =
+			_objectDefinitionLocalService.fetchObjectDefinitionByClassName(
+				TestPropsValues.getCompanyId(), Organization.class.getName());
+
+		_addObjectRelationship(
+			_objectDefinition2, systemObjectDefinition,
+			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
+
+		ObjectEntry objectEntry1 = _addObjectEntry(
+			_objectDefinition2.getObjectDefinitionId(), Collections.emptyMap());
+
+		_assertGetRelatedModels(objectEntry1.getObjectEntryId(), 0);
+
+		_objectEntryLocalService.insertIntoOrUpdateExtensionTable(
```

Comment:
This call is made 9x, I think you can move to a private method and just pass the relevant variables.